src/components/box/box.stories.mdx

```diff
@@ -63,11 +63,37 @@ Basic component that gives access to a number of styling options. Should be used
   </Story>
 </Preview>
 
-## Props
+## Scroll
+Use the `scrollVariant` prop to indicate which theme scrollbar you would like to display. Use `light` or `dark` values to display as below. This will still require the use of the `overflow` prop and only displays in Chrome or Safari
+<Preview>
+  <Story name="scroll" parameters={{ info: { disable: true }}} >
```

Comment:
I checked chromatic and the scroll bar isn't visible in the preview. I think we can disable the story unless we can find a way to fix it. We should contact chromatic using the in app chat, they normally reply within a few hours.

Reply:
I contacted them, I'll report back when they reply.

Reply:
> We hide scrollbars when taking snapshots so that screenshots capture the content only. Unfortunately there's no workaround for capturing scrollbars right now. I can see your use case though and will pass it on as a feature request to the team. 

Lets disable chromatic on this story.

Reply:
Disabled chromatic on that story